NSS-Day-Cohort-42/NSS-Day-Cohort-42.github.io
# fixed readme

-  Fixed the readme

## Steps to Test

```sh
git fetch --all
git checkout mp-readme
```

* Readme should be a list of contributors names now
